commit 348a1462aada65ac51c32850e8b94d7f7153f973
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2019-04-02T21:46:16-07:00 (6 months ago)
    
    TRITON-1365 server-setup job should not fail on first ETag conflict when marking server as setup
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/lib/workflows/server-setup.js b/lib/workflows/server-setup.js
index f50e4f5..ca85e0a 100644
--- a/lib/workflows/server-setup.js
+++ b/lib/workflows/server-setup.js
@@ -60,6 +60,7 @@ function markServerAsSettingUp(job, callback) {
     var serverUrl = '/servers/' + job.params.server_uuid;
 
     var payload = {
+        etag_retries: 3,
         setting_up: true
     };
 
@@ -239,6 +240,7 @@ function markServerAsSetup(job, callback) {
     var serverUrl = '/servers/' + job.params.server_uuid;
 
     var payload = {
+        etag_retries: 3,
         setup: true,
         setting_up: false
     };
diff --git a/package.json b/package.json
index b7030e9..30ce103 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.22.0",
+  "version": "1.22.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
